docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+services:
+    prest:
+        build: .
+        links: 
+            - "postgres:postgres"
+        environment: 
+            - PREST_PG_HOST=postgres
+            - PREST_PG_USER=postgres
+            - PREST_PG_DATABASE=prest
+        ports: 
+            - "3000:3000"
+    postgres:
+        image: postgres:9.4
+        volumes: 
+            - "./data/postgres:/var/lib/postgresql/data"
```

Comment:
Used `data` folder
This container only runs a service (focus on postgresql), don't run nothing but postgresql so don't create the internal folder